h2store/H2STORE_GW.DATA

```diff
@@ -0,0 +1,186 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 NORCE
+
+-- Test case for gas + water + disgasw (dissolved gas in water phase)
+
+----------------------------------------------------------------------------
+RUNSPEC
+----------------------------------------------------------------------------
+TITLE
+   H2STORE GAS-WATER DISGASW
+
+DIMENS 
+20 1 20 /
+
+EQLDIMS
+/
+
+TABDIMS
+/
+
+WATER
+GAS
+H2STORE
+DISGASW
+
+--DISGAS
+--DIFFUSE
+--THERMAL
+
+METRIC
+
+START
+   1 'JAN' 2019 /
+
+WELLDIMS
+   2 10 1 2 /
+
+UNIFIN
+UNIFOUT
+
+----------------------------------------------------------------------------
+GRID
+----------------------------------------------------------------------------
+
+NOECHO
+
+INIT
+
+DX 
+  400*5 /
+
+DY 
+  400*100 /
+
+DZ 
+  400*5 /
+
+TOPS
+  20*0.0 /
+
+PORO 
+  400*0.3/
+
+PERMX 
+  400*2000/
+
+PERMY
+  400*2000/
+  
+PERMZ 
+  400*2000/
+----------------------------------------------------------------------------
+PROPS
+----------------------------------------------------------------------------
+
+ROCK
+  1.0 1e-6 /
+  
+SGFN
+0	0	0.0
+1	1 	0.0 /
+
+SWFN
+0	0	0.0
+1	1 	0.0 /
+
+SALINITY
+ 0.7/ -- 35-40g/l  -> 35-40g /kg -> 0.63-0.72 mol/g
+
+----------------------------------------------------------------------------
+SOLUTION
+---------------------------------------------------------------------------
+
+
+PRESSURE
+	400*200.0 /
+SWAT
+	400*1.0 /
+
+-- Note: EQUIL is not yet supported for GAS + WATER case
+--EQUIL
+-- 0 200 1000 0 0 0 0 0 0 /
+    
+RTEMPVD
+  0 50
+  100 50
+/
+
+--RSVD
+--  0 0.0
+--  100 0.0 /
+
+RPTRST
+ 'BASIC=2' 'ALLPROPS' BW VISC KRW/
+	
+----------------------------------------------------------------------------	
+SUMMARY
+----------------------------------------------------------------------------
+
+BGSAT
+  10 1 1 /
+  10 1 10 /
+/
+
+BPR
+  10 1 1 /
+  10 1 10 /
+/
+
+BRS
+  10 1 1 /
+  10 1 10 /
+/
+
+FWIP
+FGIP
+FWPR
+FGIR
+
+WGPR
+/
+WGPRF
+/
+WGPRS
+/
+----------------------------------------------------------------------------
+SCHEDULE
+----------------------------------------------------------------------------
+
+RPTRST
+ 'BASIC=2' 'ALLPROPS' BW VISC KRW/
+
+WELSPECS
+-- Item #: 1	 2	3	4	5	 6
+	'INJ'	'G1'	1	1	1*	'GAS' /
+	'PROD'	'G1'	20	1	1*	'WATER' /
+/
+COMPDAT
+-- Item #: 1	2	3	4	5	6	7	8	9
+	'INJ'	1	1	20	20	'OPEN'	1*	1*	0.5 /
+	'PROD'	20	1	1	1	'OPEN'	1*	1*	0.5 /
+/
+
+WCONINJE
+-- Item #:1	 2	 3	 4	5      6  7
+	'INJ'	'GAS'	'OPEN'	'RATE'	100000 1* 400 /
+/
+
+WCONPROD
+-- Item #:1	2      3     4	   5     9
+	'PROD' 'OPEN' 'BHP' 1*     1* 3* 190 /
+/
+
+-- 30 day
+TSTEP
+100*1
```

Comment:
Is it necessary to have a longer run for this case in particular? All other cases are 30*1.